modules/rosa-hcp/vars.tf

```diff
@@ -41,10 +41,23 @@ variable "offline_access_token" {
 
 variable "availability_zones_count" {
   type        = number
-  description = "The number of availability zones to use for the cluster (minimum 2)"
+  description = "The count of availability (minimum 2) zones to utilize within the specified AWS Region, where pairs of public and private subnets will be generated. Valid only when availability_zones variable is not provided. This value should not be updated, please create a new resource instead."
   default     = 2
 }
 
+variable "availability_zones" {
+  type        = list(string)
+  description = "A list of availability zones names in the region. This value should not be updated, please create a new resource instead"
+  default     = null
```

Comment:
why wouldn't the default also be `[]` for this `list(string)`?

Reply:
if it differs from the default assumption of a `list(string)` and another is being used under it with a different value.
We should probably either align or add a comment on why it differs.